deploy/crds/ocs_v1_storagecluster_crd.yaml

```diff
@@ -107,6 +107,8 @@ spec:
                 - dataPVCTemplate
                 type: object
               type: array
+            useHostPathForMons:
```

Comment:
Do we allow the host path to be specified? For example, `/var/lib/rook` might not always be valid. Instead of a bool, should this just be a string for the host path? Or is it possible to specify somewhere else?

Reply:
yup the hostPath is still there. This PR just adds a boolean flag to disable use of pvcTemplate when enabled.

Reply:
yup the hostPath is still there. This PR just adds a boolean flag to disable use of pvcTemplate when enabled. https://github.com/openshift/ocs-operator/pull/410/files#diff-24bbbd5f2f20ffccf914757966c7401aR765